src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/view-data-brokers/page.tsx

```diff
@@ -27,8 +33,17 @@ export default async function ViewDataBrokers() {
   const scanResult = await getLatestOnerepScan(profileId);
   const scanResultItems = scanResult?.onerep_scan_results?.data ?? [];
 
+  // TODO: Find a better way to get totalExposures
+  const subBreaches = await getSubscriberBreaches(session.user);
+  const summary = dashboardSummary(scanResultItems, subBreaches);
+
   // TODO: Use api to set/query count
   const countOfDataBrokerProfiles = scanResultItems.length;
+  const estimatedTime = countOfDataBrokerProfiles * 10; // 10 minutes per data broker site.
```

Comment:
Thanks for this comment, I was just looking for where I could find that value :)